pkg/node/netconfig.go

```diff
@@ -31,6 +31,7 @@ const (
 	componentNetworkConfiguration = "network-configuration"
 	// Interface name where to add service IP
 	loopbackInterface = "lo"
+	ovnExternalBridge = "br-ex"
```

Comment:
There is another const in pkg/config/ovn/ovn.go for the "br-ex", can we reuse it or define it in the ovn.go file if the ovnGatewayInterface is going to be removed?
https://github.com/openshift/microshift/blob/main/pkg/config/ovn/ovn.go#L17

Reply:
Yes